scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Hardcoded API key detected. This is a critical security risk[^1]. Never store sensitive information like API keys directly in the source code.

Recommendation: Use environment variables or a secure secret management system to store and retrieve sensitive information.
```suggestion
const apiKey = process.env.API_KEY // Retrieve API key from environment variable
```

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html